doc/source/release-actions/index.rst

```diff
@@ -31,3 +31,53 @@ Release GitHub action
 .. jinja:: release-github
     :file: _templates/action.rst.jinja
 
+Release PyPI template
+---------------------
+
+Below, it is assumed that a job `build-library` already created library sources
+and wheel artifacts using the environment variable `PACKAGE_NAME`. Also, the job
+is defined to not upload files if one already exists thanks to the
+`skip-existing` input.
```

Comment:
```suggestion
The following code assumes that a `build-library` job has already created library sources
and wheel artifacts using the `PACKAGE_NAME` environment variable. Also, the job's 
`skip-existing` input indicates that files that already exist are not uploaded again.
```
It's clearer when the noun describing the code entity follows the code entity. Also, we should avoid placement indicators like above and below and use preceding and following instead. This is because a PDF of the documentation might have placed content on a preceding or following page.